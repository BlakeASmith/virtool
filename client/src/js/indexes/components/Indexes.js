import React from "react";
import { Route, Switch } from "react-router-dom";
import IndexesList from "./List";

const Indexes = () => (
    <Switch>
        <Route path="/refs/:refId/indexes" component={IndexesList} exact />
    </Switch>
);

export default Indexes;
