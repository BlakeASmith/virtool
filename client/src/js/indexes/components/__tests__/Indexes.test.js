import Indexes from "../Indexes";

describe("<Indexes />", () => {
    it("should render", () => {
        const wrapper = shallow(<Indexes />);
        expect(wrapper).toMatchSnapshot();
    });
});
