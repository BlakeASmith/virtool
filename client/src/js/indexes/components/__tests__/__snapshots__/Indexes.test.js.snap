// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Indexes /> should render 1`] = `
<Switch>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(IndexesList)",
        "type": [Function],
      }
    }
    exact={true}
    path="/refs/:refId/indexes"
  />
</Switch>
`;
